app/lib/efile/nj/nj1040.rb

```diff
@@ -55,6 +56,19 @@ def analytics_attrs
 
       private
 
+      def calculate_line_15
+        if @direct_file_data.w2s.empty?
+          return -1
+        end
+
+        sum = 0
+        @direct_file_data.w2s.each do |w2|
+          state_wage = w2.node.at("W2StateLocalTaxGrp StateWagesAmt").text.to_f
```

Comment:
[PEBBLE] you can use the accessor on `DfW2` here https://github.com/codeforamerica/vita-min/blob/main/app/models/df_w2_accessor.rb#L18

so it would look like `state_wage = w2.StateWagesAmt.to_f`

Reply:
Replied in slack, agreed we're not changing for now